atlp-rwanda/ecommerce-app-legends-fe
# #59 Implement sellers product deletion functionalities

#### What does this PR do?

This pull request implements the feature to allow sellers to delete a specific item in their collection.

#### Description of Task to be completed?

The task involved implementing a delete functionality for items in a seller's collection. The feature includes a confirmatory dialogue that appears when the seller clicks on the delete button. If confirmed, the item will be permanently deleted. Additionally, I have also implemented a product lists page for the vendor's collection.

#### How should this be manually tested?

To test this feature manually, please follow the steps below:

1. Login to the seller's account. `Need to be added by Admin by this moment this PR is being raised`
2. Navigate to the products page.
3. Click on the delete button (bin Icon).
4. Confirm the dialogue that appears.
5. Verify that the item has been deleted from the collection.

#### Any background context you want to provide?

No background context is necessary for this pull request.

#### What are the relevant pivotal tracker/Trello stories?

This pull request is related to Trello card #59.

#### Screenshots (if appropriate)

<img width="480" alt="image" src="https://github.com/atlp-rwanda/ecommerce-app-legends-fe/assets/46238001/5aa86f24-ae6a-49ee-993b-64b421ca0c36">
#### Ploduct Lists

<img width="480" alt="image" src="https://github.com/atlp-rwanda/ecommerce-app-legends-fe/assets/46238001/b8467243-de87-45c7-81e7-03c0c377c01b">
Prompt to comfirm deletion

<img width="480" alt="image" src="https://github.com/atlp-rwanda/ecommerce-app-legends-fe/assets/46238001/c69e03f1-31c0-431a-acd0-9012f730b56d">
Comfirmed

#### Questions:

If you have any questions or comments regarding this pull request, please let me know.
